packages/gatsby/src/services/start-webpack-server.ts

```diff
@@ -0,0 +1,117 @@
+import openurl from "better-opn"
+import report from "gatsby-cli/lib/reporter"
+import formatWebpackMessages from "react-dev-utils/formatWebpackMessages"
+import chalk from "chalk"
+import { Compiler } from "webpack"
+
+import {
+  reportWebpackWarnings,
+  structureWebpackErrors,
+} from "../utils/webpack-error-utils"
+
+import { printDeprecationWarnings } from "../utils/print-deprecation-warnings"
+import { printInstructions } from "../utils/print-instructions"
+import { prepareUrls } from "../utils/prepare-urls"
+import { startServer } from "../utils/start-server"
```

Comment:
bikesheddy: given how all those utils are only used by this service, could we group them in some directory and not just add 4 more files to `utils`?

so maybe:

```
 - services/
    - start-webpack-server/
       - index.ts (this file)
       - print-deprecation-warnings (utils - repeat for other util)
```
```

Reply:
I like the cut of your jib